doc/update032.md

```diff
@@ -0,0 +1,155 @@
+### dPoW 0.3.2 update instructions
+
+- Clone and Build GleecBTC in your 3p node. Checkout to the commit `afc1ab4`
```

Comment:
there is different commit on https://github.com/KomodoPlatform/dPoW/pull/206/files#diff-635c287155ae8c74a385414a7b871ecb161702ee294fe7f7ddf70d2f7408ed22R11 and in Readme